common/types/question.d.ts

```diff
@@ -0,0 +1,14 @@
+interface Question {
+    id: string,
+    title: string,
+    complexity: string,
+    description?: string,
```

Comment:
The way I see this `Question` type is different as yours. I will think that `description` is necessary so that users know what is the question about. I also think that `category` and `url` are necessary too. Is it okay if I restructure this type when we restructure the backend?

Reply:
the attributes are made optional for my dummy backend only, feel free to change them.

Reply:
Got it, thanks for explaining!